bundles/org.openhab.binding.nibeheatpump/src/main/resources/OH-INF/thing/SMO40.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="nibeheatpump"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="smo40-udp">
+		<label>UDP Connected Nibe SMO40 Heat Pumps</label>
+		<description></description>
+
+		<channel-groups>
+			<channel-group typeId="smo40-sensor-group-channels" id="sensor"/>
+			<channel-group typeId="smo40-setting-group-channels" id="setting"/>
+		</channel-groups>
+
+		<config-description>
+			<parameter name="hostName" type="text" required="true">
+				<label>Host Name</label>
```

Comment:
You could specify `<context>network-address</context>` to get a free validation.

Reply:
The code of SMO40.xml is equal to the code uf d1x45.xml and 1x55.xml.
I suggest to change all classes and keep them equal in their behaviour